openstack/sharedfilesystems/v2/shares/results.go

```diff
@@ -144,3 +144,36 @@ func (r GetExportLocationsResult) Extract() ([]ExportLocation, error) {
 	err := r.ExtractInto(&s)
 	return s.ExportLocations, err
 }
+
+// AccessRight contains all information associated with an OpenStack share
+// Grant Access Response
+type AccessRight struct {
+	// The UUID of the share to which you are granted or denied access.
+	ShareID string `json:"share_id"`
+	// The access rule type that can be "ip", "cert" or "user".
+	AccessType string `json:"access_type,omitempty"`
```

Comment:
@jrperritt General question: Does `omitempty` provide any use here?

If I do:

```shell
$ grep -R omitempty ~/go/src/github.com/gophercloud/gophercloud | grep result
```

I see that there are a good number of results.